### ISSUE TITLE
Нужен исполняемый файл под linux для распаковки hbk

### ISSUE THREAD

**Issue by tormozit** [xDrivenDevelopment/precommit1c#161]
Извиняюсь что не совсем по продукту. 
Решил тут линукс поддержать в ИР, не устанавливая сам линукс). Дошел до консольной утилиты v8unpack, с помощью которой у меня выполняется распаковка файлов синтакс-помощника (hbk). Есть ли у вас сборка этой утилиты или ее аналога для linux?

**Comment by EvilBeaver** [comment]
@dmpas закончилась чем-либо эпопея с портированием инструментов под линукс?

**Comment by tormozit** [comment]
@EvilBeaver Если ты про ИР, то 90% функционала уже работает на Linux.

**Comment by EvilBeaver** [comment]
@tormozit нет, я же призвал Сергея (@dmpas) и имел в виду их работу по портированию всех инструментов Валеры Агеева (включаяя v8unpack) на линукс. Я не знаю насколько она завершена.

**Comment by dmpas** [comment]
@EvilBeaver порт есть. Обсуждаем с @tormozit  варианты работы без установки

**Comment by tormozit** [comment]
Да. Обсуждаем тут https://github.com/e8tools/v8unpack/issues/68
